src/status_im/group_chats/core.cljs

```diff
@@ -79,11 +88,11 @@
 (fx/defn add-members
   "Add members to a group chat"
   {:events [:group-chats.ui/add-members-pressed]}
-  [{{:keys [current-chat-id selected-participants]} :db :as cofx}]
+  [{{:keys [current-chat-id]} :db :as cofx}]
   {::json-rpc/call [{:method      "wakuext_addMembersToGroupChat"
-                     :params      [nil current-chat-id selected-participants]
+                     :params      [nil current-chat-id (get-in cofx [:db :group-chat/selected-participants])]
```

Comment:
you could use namespaced keys destructuring

Reply:
`:group-chat/keys [selected-participants]`

Reply:
Thank you. Fixed.